code/modules/projectiles/projectile/special/crystal.dm

```diff
@@ -0,0 +1,31 @@
+/obj/projectile/crystal
+	name = "crystal"
+	icon_state = "crystal_square"
+	pass_flags = PASSTABLE | PASSGLASS | PASSGRILLE
+	damage = 200 // dont get hit by this
+	light_range = 2
+	damage_type = BURN
+	hitsound = 'sound/weapons/sear.ogg'
+	hitsound_wall = 'sound/weapons/effects/searwall.ogg'
+	flag = "laser"
+	eyeblur = 2
+	impact_effect_type = /obj/effect/temp_visual/impact_effect/red_laser
+	light_color = LIGHT_COLOR_RED
+	ricochets_max = 50
+	ricochet_chance = 80
+	reflectable = REFLECT_NORMAL
+	var/is_test_fire = TRUE
+	var/which_emitter = 1
+	var/crystal_color = CRYSTAL_COLOR_RED
+	var/crystal_size = CRYSTAL_SIZE_MEDIUM
+	var/crystal_shape = CRYSTAL_SHAPE_SQUARE
+
+
+/obj/projectile/crystal/on_hit(atom/target)
+	. = ..()
+	if(iscarbon(target)) // dont get hit by this
```

Comment:
hm yes this is balanced and traitors definitely won't abuse this

similarly, epic gamers DEFINITELY won't abuse the base 200 damage of this projectile to rip apart blob tiles

Reply:
also, is it intentional that borgs and simplemobs don't get dusted by this?

Reply:
what did you expect? well thought out design?

Reply:
> what did you expect? well thought out design?

I fail to see how what I EXPECTED the design quality to be would be relevant here. What matters is what the design quality IS NOW and how we can improve it.

Reply:
What did you expect? Sportiflop?

Reply:
you can't move the emitters at all, they're stationary

Reply:
But you can build them, no? You don't need to move these emitters around to make a trap or a blob-killing device with them.

Also, even if you can't build or move these in any way, you can still use reflectors to bounce the beams from them around to where you want them to be for your trap/cheap instakill device.

Reply:
oh god reflectors